src/pages/story/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components'
+import { OurStory } from './_our-story'
+import { Container, SEO, Show } from 'components/containers'
+import { getWindowWidth, isBrowser } from 'common/utility'
+import Layout from 'components/layout/layout'
+import { localize, WithIntl } from 'components/localization'
+import { Header, Text } from 'components/elements'
+import device, { size } from 'themes/device'
+
+const Background = styled.div`
+    background: var(--color-black);
+    width: 100%;
+    height: 100%;
+`
+
+const StyledContainer = styled(Container)`
+    padding: 12rem 19rem;
+    white-space: nowrap;
+
+    @media ${device.laptopL} {
+        padding: 12rem 4rem;
+    }
+    @media ${device.tabletL} {
+        padding: 5rem 2rem;
+        width: 100%;
+
+        > h1 {
+            font-size: 4.5rem;
+        }
+    }
+`
+const ContentWrapper = styled.div`
+    margin-top: ${(props) => props.margin_top || 'none'};
+    white-space: normal;
+    max-width: 79.8rem;
+
+    @media ${device.mobileL} {
+        max-width: 42.8rem;
+    }
+`
+
+const Story = () => {
+    const [is_mobile, setMobile] = useState(false)
+
+    useEffect(() => {
+        setMobile(isBrowser() && getWindowWidth() <= size.tablet)
+    }, [getWindowWidth()])
+
+    return (
+        <Layout>
+            <SEO
+                title={localize('About Us | An Online Trading Platform | Deriv.com')}
+                description={localize(
+                    'Deriv.com - A Binary.com brand, is a pioneer and award-winning online trading platform in the trading market.',
+                )}
+            />
+            <Background>
+                <StyledContainer direction="column">
+                    <Header as="h1" type="display-title" color="white" align="center">
+                        {localize('Our Story')}
```

Comment:
```suggestion
                        {localize('Our story')}
```

Reply:
This should be small caps to match the previous string used, so that the translations are able to detect it.